docs/grammar.txt

```diff
@@ -68,7 +68,8 @@ Continue = 'continue'
 Break = 'break'
 Return = 'return' Expression?
 Throw = 'throw'
-VariableDefinition = VariableDeclaration ( '=' Expression )?
+VariableDefinition = ('var' DeclarationList | VariableDeclaration) ( '=' Expression )?
+DeclarationList = '(' ( Identifier ',' | ',' )* ( Identifier ','? | ',' ) ')'
```

Comment:
Why don't you simplify it to 
`DeclarationList = '(' ( Identifier? ',' )* Identifier? ')'`